llvm/test/tools/llvm-cas-object-format/debug_line_split.test

```diff
@@ -0,0 +1,45 @@
+RUN: rm -rf %t && mkdir -p %t
+RUN: llc %S/Inputs/debug_line_split.ll --filetype=obj -o %t/debug_line_split.o
+
+RUN: llvm-cas-object-format --cas %t/cas --dump --ingest-schema=nestedv1 %t/debug_line_split.o | FileCheck %s
+
+CHECK: section __DWARF,__debug_line:
+CHECK-EMPTY: 
+CHECK-NEXT:   block 0x4d8 size = 0x00000046, align = 1, alignment-offset = 0
```

Comment:
We probably shouldn't hardcode any offsets and sizes here because they will change with every update anywhere in the LLVM optimization pipeline.